schemas/compat-data-schema.md

```diff
@@ -226,6 +226,17 @@ Examples:
 }
 ```
 
+### Ranged versions
+
+For certain browsers, ranged versions are allowed as it is sometimes impossible to find out in which early version of a browser a feature shipped. The statement below means "supported in at least version 37 and probably in earlier versions as well".
+Currently, the only allowed ranged version is `"≤37"` for `webview_android`. There will be more ranged versions for other browsers in the future but ranged versions aren't generally allowed for every version string. Ranged versions should be used sparingly and only when it is impossible to find out the version number a feature initially shipped in.
```

Comment:
Maybe we should move this line to the data guidelines? Not a blocker, but something to consider

Reply:
I think, for the initial launch of ranged versions, it would be nice to have the information in one place. If we're allowing more ranged versions for other browsers/versions, we could surely move and expand this to the data guideline doc.